src/genImage.go

```diff
@@ -0,0 +1,62 @@
+package src
+
+import (
+	"fmt"
+	"image"
+	color "image/color"
+	"image/jpeg"
+	"log"
+	"os"
+	"strconv"
+)
+
+var x = 0
+var y = 0
+var width = 400
+var height = 300
```

Comment:
```suggestion
const (
    x = 0
    y = 0
    width = 100
    height = 100
)
```
constにできます。あと画像サイズもっと小さくても良いかも。suggestでは小さくしてます。